travis.sh

```diff
@@ -322,6 +359,7 @@ test ${WARNINGS_OK:=true} == true -o "$WARNINGS_OK" == 1 -o "$WARNINGS_OK" == ye
 travis_run --title "CXX compiler info" $CXX --version
 
 update_system
+setup_ssh_keys
```

Comment:
Why running setup_ssh_keys twice? I think the call here is sufficient.

Reply:
You're totally right, so right now we have these two solutions

1- [Solution#1](https://github.com/JafarAbdi/moveit_ci/pull/4/files)
  - We pass the ssh credentials to the docker and set up a new ssh-agent there

2- [Solution#2](https://github.com/JafarAbdi/moveit_ci/pull/3/files/bf4d7af0d773e24e086cf8ed131e5f808d46ac79)
  - We use the ssh-agent started inside Travis

Which one do you think we should use .?

Reply:
I think, only solution#2 is feasible. See the comment I left for solution#1.

Reply:
I tested both solutions on a private repo and both of them are working

Merging the solution#2 fails the Travis here, seems like Travis don't start ssh-agent which cause this error `SSH_AUTH_SOCK: unbound variable` [see](https://travis-ci.org/ros-planning/moveit_ci/jobs/609293497?utm_medium=notification&utm_source=github_status)

The solution#1 should work, too, because setting up ssh-agent inside the docker will set new a `SSH_AUTH_SOCK` and `SSH_AGENT_PID` using the copied ssh keys